imbi/endpoints/project_ssm_configuration.py

```diff
@@ -61,10 +61,13 @@ async def get(self, *args, **kwargs) -> None:
                 project_info['project_slug'],
                 title='Wrong configuration type for project')
 
-        env = os.environ.get('ENVIRONMENT', 'production').lower()
-        google_integration = await oauth2.OAuth2Integration.by_name(
-            self.application, f'google-{env}')
-        tokens = await google_integration.get_user_tokens(self.current_user)
+        if not self.settings['google']['integration_name']:
```

Comment:
We should also raise an exception if the google integration is disabled by configuration.